.gitignore

```diff
@@ -2,4 +2,12 @@
 node_modules/
 
 # IDEA
-.idea/
\ No newline at end of file
+.idea/
+
+# Visual Studios
+.trunk/
+
+# MOJ Express Skeleton
+public/images/*
+public/javascripts/*
+public/stylesheets/*
```

Comment:
Do we need anything from public folder ? May be we can ignore anything from public/* ?

Reply:
Well, I don't think we generate the public folder when we need to use it, so I just excluded the content that we create. However we could tweak this so it does create a public folder when we do a `npm run`